src/lib/bootloader/grub2_widgets.rb

```diff
@@ -472,6 +472,19 @@ def init
       Yast::UI.ChangeWidget(Id(:theme), :Value, grub_default.theme || "")
     end
 
+    def validate
+      if Yast::UI.QueryWidget(Id(:console_frame), :Value)
+        console_value = Yast::UI.QueryWidget(Id(:console_args), :Value)
+        if console_value.strip.empty?
+          Yast::Report.Error(
+            _("To enable serial console you must provide the corresponding arguments.")
+          )
```

Comment:
good habit in validation is to focus problematic widget `Yast::UI.SetFocus(:console_args)`